code/modules/projectiles/guns/energy/nuclear.dm

```diff
@@ -51,6 +52,8 @@
 	origin_tech = list(TECH_COMBAT = 3, TECH_MATERIAL = 5, TECH_POWER = 3)
 	slot_flags = SLOT_BELT
 	force = 8 //looks heavier than a pistol
+	w_class = 4	//Looks bigger than a pistol, too.

```

Comment:
But this makes it as heavy as a shotgun. This can't be correct. https://github.com/PolarisSS13/Polaris/blob/fa74226e9901e080d762ed01dc5e6b7756cc27f1/code/modules/projectiles/guns/projectile/shotgun.dm#L7